.kafka/docker-compose-single-kafka.yml

```
@@ -0,0 +1,18 @@
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:7.3.10
+    environment:
+        ZOOKEEPER_CLIENT_PORT: 2181
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)